Packs/Slack/Integrations/Slack/Slack.yml

```diff
@@ -51,10 +51,10 @@ configuration:
 - defaultvalue: 'true'
   display: Long running instance. Required for investigation mirroring and direct
     messages.
+  hidden: true
```

Comment:
lets remove any mention of this param in the readme

Reply:
There is no mention of it in the readme